client/apiclient_test.go

```diff
@@ -11,6 +11,8 @@ import (
 type mockHTTP struct {
 	called []*http.Request
 	resp   string
+
+	Transport *http.Transport
```

Comment:
do you have any plans to add unit tests for our new code paths?  our functional tests should capture the actual, erm, functionality, but unit tests are quite nice too.

Reply:
We have a work item on the internal backlog now to add some unit tests, and we should be getting to that issue very soon (prior to release).